integrationtest/prerelease/data/mail-v1.0.0-rc2.txt

```diff
@@ -0,0 +1,58 @@
+First line is skipped
+Subject: [MajorRelease] sandbox v1.0.0-rc2
+
+
+
+v1.0.0-rc2
```

Comment:
Oh, OK, I see you repeat the original release notes from the first rc. In the future I think it would make sense to treat following rcs as patch releases and only list bug fixes and only repeat the full release notes on the final release. But again, for a following PR.